laser/laser_class.py

```diff
@@ -0,0 +1,14 @@
+import RPi.GPIO as GPIO
+
+class Laser:
+    def __init__(self, laser_pin):
+        #Pin Setup
+        self.laser_pin = laser_pin
```

Comment:
```suggestion
        self._laser_pin = laser_pin
```
Private members in Python are generally indicated with an underscore.